### PR TITLE
[Maps - Render & Spatial Services] add v2022-08-01

### LINKED ISSUE #19535: [Maps - Render & Spatial Services] REST API Review
Service name: Azure Maps Render & Spatial Services
Key contact for the review: Gigi Grajo [gigigrajo@microsoft.com](mailto:gigigrajo@microsoft.com) 
Whether this is a new or existing API: new version of an existing API
Whether or not your service already been deployed: not deployed
A brief description of the material to be reviewed/changed: New version of Render API and Spatial API in Azure Maps

### PR DESCRIPTION
**Pull request by gigigoo0**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Add a changelog entry for this PR by answering the following questions:
  1. What's the purpose of the update?
      - [ ] new service onboarding
      - [x] new API version
      - [ ] update existing version for new feature
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify
  2. When are you targeting to deploy the new service/feature to public regions? Please provide the date or, if the date is not yet available, the month. _December 2022_
  4. When do you expect to publish the swagger? Please provide date or, the the date is not yet available, the month. _December 2022_
  5. If updating an existing version, please select the specific language SDKs and CLIs that must be refreshed after the swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [x] No refresh required for updates in this PR

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### Breaking Change Review Checklist 
If any of the following scenarios apply to the PR, request approval from the Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 
- [ ] Removing API(s) in a stable version
- [ ] Removing properties in a stable version
- [ ] Removing API version(s) in a stable version
- [ ] Updating API in a stable or public preview version with Breaking Change Validation errors
- [x] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/maps/data-plane/Spatial/stable/2022-12-01/examples/Spatial_EvaluatePointInPolygon.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "parameters": {
+    "geography": "us",
+    "format": "json",
+    "api-version": "1.0",
+    "udid": "[udid]",
```
Comment (danielthank) [inline specification/maps/data-plane/Spatial/stable/2022-12-01/examples/Spatial_EvaluatePointInPolygon.json:6]: How about using a real udid in these examples?

File: specification/maps/data-plane/Spatial/stable/2022-12-01/spatial.json [hunk 2]
```diff
@@ -0,0 +1,924 @@
...
+      "scopes": {
+        "https://atlas.microsoft.com/.default": "https://atlas.microsoft.com/.default"
+      }
+    },
+    "SharedKey": {
```
Comment (danielthank) [inline specification/maps/data-plane/Spatial/stable/2022-12-01/spatial.json:33]: `AzureKey`

File: specification/maps/data-plane/Spatial/stable/2022-12-01/spatial.json [hunk 3]
```diff
@@ -0,0 +1,924 @@
...
+    }
+  },
+  "security": [
+    {
+      "AADToken": [
+        "https://atlas.microsoft.com/.default"
+      ]
+    },
+    {
+      "SharedKey": []
```
Comment (danielthank) [inline specification/maps/data-plane/Spatial/stable/2022-12-01/spatial.json:53]: `AzureKey`

File: specification/maps/data-plane/Spatial/stable/2022-12-01/spatial.json [hunk 4]
```diff
@@ -0,0 +1,924 @@
...
+  ],
+  "responses": {},
+  "parameters": {
+    "ApiVersion": {
+      "name": "api-version",
+      "description": "Version number of Azure Maps API.",
+      "type": "string",
+      "in": "query",
+      "required": true,
+      "default": "2022-12-01",
```
Comment (danielthank) [inline specification/maps/data-plane/Spatial/stable/2022-12-01/spatial.json:67]: spectral: A required parameter should not specify a default value.

File: specification/maps/data-plane/Spatial/stable/2022-12-01/spatial.json [hunk 5]
```diff
@@ -0,0 +1,924 @@
...
+        "operationId": "Spatial_EvaluatePointInPolygon",
```
Comment (danielthank) [inline specification/maps/data-plane/Spatial/stable/2022-12-01/spatial.json:573]: Other APIs use `Get` and `Post` in the title. And just this one uses `Evaluate`. How about changing it to `GetPointInPolygon`.
spectral validation: OperationId for get method should contain "Get" or "list"

**Comment by gigigoo0** [comment]
@tjprescott Do we need to resolve checks from [~[Staging] Swagger LintDiff](https://github.com/Azure/azure-rest-api-specs/pull/19520/checks?check_run_id=7332414281) check?

**Review by tjprescott — CHANGES_REQUESTED** [review]
This needs to be restructured to facilitate review. 
Commit 1: Copy old Swagger to new folder
Commit 2: Change APIVersion from old value to new one.
Commit 3: Actual API changes from the old version to the new.

**Review by tjprescott — APPROVED** [review]

**Pull request merged**